Added a troubleshooting step for DNS issues with manifest downloads (#8749)
fishtown-analytics/docs.getdbt.com: website/snippets/_fusion-troubleshooting.md

This should resolve the error and open your dbt project by opening the workspace it belongs to. For more information on workspaces, refer to [What is a VS Code workspace?](https://code.visualstudio.com/docs/editing/workspaces/workspaces).
</Expandable>

<Expandable alt_header="Manifest cannot be downloaded from the dbt platform">

> If the **dbt Extension** cannot download the manifest from the **dbt platform** or you get `warning: dbt1200: Failed to download manifest` using **Fusion CLI**, you are probably having DNS related issues.

Reviewer: ```suggestion
If the dbt VS Code extension cannot download the manifest from the <Constant name="dbt_platform" /> or you get `warning: dbt1200: Failed to download manifest` using <Constant name="fusion" /> locally, you are probably having DNS-related issues.
```

> To confirm this you can do DNS lookup for the host dbt fusion is trying to download from (e.g. prodeu2.blob.core.windows.net) by using `dig` on Linux or `nslookup` on Windows.

Reviewer: ```suggestion
To confirm this, do a DNS lookup for the host <Constant name="fusion" /> is trying to download from (for example, prodeu2.blob.core.windows.net) by using `dig` on Linux/Mac or `nslookup` on Windows.
```

> If this won't return an IP address, the probable reason is you company is using the same cloud provider with private endpoints for cloud resources and the DNS requests for these are forwarded to private DNS zones.

Reviewer: ```suggestion
If this doesn't return an IP address, the likely reason is that your company uses the same cloud provider with private endpoints for cloud resources, and DNS requests for these are forwarded to private DNS zones.
```

> This situation can be remedied by setting up an internet fallback which will then return a public ip to any cloud storage that do not have a private ip registered with the private dns zone.

Reviewer: ```suggestion
This situation can be remedied by setting up an internet fallback, which will then return a public IP to any cloud storage that does not have a private IP registered with the private DNS zone.
```
